res/TensorFlowLiteRecipes/Net_Sqrt_Div_000/test.recipe

```diff
@@ -0,0 +1,54 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 3 dim: 2 }
+}
+# Add 100.0 is not to make divide by zero error in testing
+operand {
+  name: "one"
+  type: FLOAT32
+  shape { dim: 1 }
+  filler {
+    tag: "constant" arg: "100.0"
+  }
+}
+operand {
+  name: "add"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 3 dim: 2 }
+}
+operand {
+  name: "sqrt"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 3 dim: 2 }
+}
+operand {
+  name: "div"
+  type: FLOAT32
+  shape { dim: 1 dim: 3 dim: 3 dim: 2 }
+}
+operation {
+  type: "Add"
+  input: "ifm"
+  input: "one"
+  output: "add"
+  add_options {
+    activation: NONE
+  }
+}
+operation {
+  type: "Sqrt"
+  input: "add"
+  output: "sqrt"
+}
+operation {
+  type: "Div"
+  input: "ifm"
+  input: "sqrt"
+  output: "div"
+  div_options {
+    activation: NONE
+  }
+}
+input: "ifm"
```

Comment:
Is it your intention to have only one input `ifm`, and share it with `Div` and `Add`.

Reply:
Yup :)